pkg/components/rook/component.go

```diff
@@ -40,6 +40,7 @@ type component struct {
 	AgentTolerationEffect    string `hcl:"agent_toleration_effect,optional"`
 	DiscoverTolerationKey    string `hcl:"discover_toleration_key,optional"`
 	DiscoverTolerationEffect string `hcl:"discover_toleration_effect,optional"`
+	EnableMonitoring         bool   `hcl:"enable_monitoring,optional"`
```

Comment:
I'm on the fence with `enable_monitoring` here. We use `service_monitor` for some components, `metrics` for others. `enable_monitoring` make sense here, but inconsistency is bothering.

Reply:
Created an issue for it https://github.com/kinvolk/lokomotive/issues/407.

Reply:
Eventually we should change all of them to `enable_monitoring`. We have to start somewhere.

Reply:
Still not sure as mentioned in the issue, but it seems reasonable, so it's fine for now IMO.